lib/Agrammon/OutputFormatter/Excel.pm6

```diff
@@ -5,22 +5,96 @@ use Spreadsheet::XLSX;
+            :custom-width, :width(50);
+    $output-sheet.columns[2] = Spreadsheet::XLSX::Worksheet::Column.new:
+            :custom-width, :width(32);
+    my $row = 2;
+    for sorted-kv($outputs.get-outputs-hash) -> $module, $_ {
+        my $col = 0;
+        my $n = 0;
+        when Hash {
+            for sorted-kv($_) -> $output, $value {
+                my $val = flat-value($value // 'UNDEFINED');
+                my $var-print = $model.output-print($module, $output) ~ ',All';
+                if not $prints or $var-print.split(',') ∩ @print-set {
+                    $n++;
+                    my $unit = $model.output-unit($module, $output, $language);
+                    $output-sheet.set($row, $col+0, $module);
+                    $output-sheet.set($row, $col+2, $output);
+                    $output-sheet.set($row, $col+3, $val, :number-format('#,###'));
+                    $output-sheet.set($row, $col+4, $unit);
+
+# TODO
+#                    if $include-filters {
+#                        if $value ~~ Agrammon::Outputs::FilterGroupCollection && $value.has-filters {
+#                            render-filters(@module-lines, $value, $unit, $indent, :$all-filters);
+#                        }
+#                    }
+                    $row++;
+                }
+            }
+        }
+        when Array {
+            for sorted-kv($_) -> $instance-id, %instance-outputs {
+                for sorted-kv(%instance-outputs) -> $fq-name, %values {
+                    my $q-name = $module ~ '[' ~ $instance-id ~ ']' ~ $fq-name.substr($module.chars);
+                    for sorted-kv(%values) -> $output, $value {
+                        my $val = flat-value($value // 'UNDEFINED');
+                        my $var-print = $model.output-print($module, $output) ~ ',All';
+                        if not $prints or $var-print.split(',') ∩ @print-set {
+                            $n++;
+                            my $unit = $model.output-unit($module, $output, $language);
+                            $output-sheet.set($row, $col+0, $module);
+                            $output-sheet.set($row, $col+1, $q-name);
+                            $output-sheet.set($row, $col+2, $output);
+                            $output-sheet.set($row, $col+3, $val, :number-format('#,###'));
+                            $output-sheet.set($row, $col+4, $unit);
+
+# TODO
+#                            if $include-filters {
+#                                if $value ~~ Agrammon::Outputs::FilterGroupCollection && $value.has-filters {
+#                                    render-filters(@module-lines, $value, $unit, $indent, :$all-filters);
+#                                }
+#                            }
+                            $row++;
+                        }
+                    }
+                }
+            }
+        }
+        NEXT { # empty line between sections
+            $row++ if $n;
+        }
```

Comment:
Totally fine, but in case you didn't know: for single-statement phasers you don't need the curlies and can just write `NEXT $row++ if $n`.